example.env

```diff
@@ -0,0 +1,5 @@
+DATABASE_URL=
+PGADMIN_EMAIL=
+PGADMIN_PASSWORD=
```

Comment:
I'd also add `POSTGRES_USER` and `POSTGRES_PASSWORD` and update docker-compose.yml to read it from .env file